configure.ac

```diff
@@ -614,7 +620,9 @@ dnl Check for endianness
 AC_C_BIGENDIAN
 
 dnl Check for pthread compile/link requirements
-AX_PTHREAD
+if test $host !=  *mingw*; then
+   AX_PTHREAD
+fi
```

Comment:
@Naviabheeman Bitcoin Core does not appear to have made this change. Why is this necessary?

Reply:
this was an attempt to fix the pthread flag. I think it's not necessary. I'll wait for the CI to finish and verify.